circle.yml

```diff
@@ -67,6 +67,11 @@ deployment:
     owner: palantir
     commands:
       - ./scripts/circle-ci/publish-github-page.sh
+  publish-snapshots:
+    branch: /.*/
```

Comment:
I'm concerned that this would lead to a very odd layout of artifacts.  You can imagine that after a release several branches are created and work is done resulting in several snapshots published of the format 0.x.y+g1-<hash> and then some with Z commits at 0.x.y-gZ-<hash> but unlike what you might expect they are not progressing forward in a linear fashion.  The most frustrating of which would have the leading edge develop be 0.x.y+g23-<hash> but there are likely branches with higher numbers (based on the head of develop) which would lead foolish resolves to pick those branches instead (as they appear to be more recent).  I.e. without somehow including branch name, I do think it would be a good idea to publish all of the branches.  If someone would like to have a snapshot of their branch I would instruct them to publish it manually (and preferably just locally)